campaignion_form_builder/campaignion_form_builder.type_opt_in.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Drupal\little_helpers\ArrayConfig;
+
+/**
+ * Invoked by campaignion_form_builder_form_builder_element_types().
+ */
+function _campaignion_form_builder_form_builder_types_opt_in() {
+  $fields = [];
+
+  $opt_in = function($component, $weight) {
+    $element = [
+      'title' => $component['name'],
+      'weight' => $weight,
+      'unique' => TRUE,
```

Comment:
Not sure if we want it to be unique, right now, multiple newsletter fields are possible (using campaignion_newsletters)?

Reply:
Our current interpretation of phone and post opt-ins is that they are contact-wide. So it wouldn’t make much sense to have more than one field per form, right?

Reply:
OK.